pttrack/models.py

```diff
@@ -232,6 +233,10 @@ class Patient(Person):
     preferred_contact_method = models.ForeignKey(ContactMethod, blank=True,
                                                  null=True)
 
+    # If the patient is in clinic and needs a workup, that is specified by needs_workup. Default value is false for all the previous patients
+
+    needs_workup = models.BooleanField(default=False)
```

Comment:
My gut says to make the default `True` here. That way, any newly created patient will *automatically* have this flag set. That way, you don't have to manually set it to true in `form_valid` in the view.